src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/CreateSearchableReleaseDocuments/Dtos/ReleaseVersionPublishedEventDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.CreateSearchableReleaseDocuments.Dtos;
+
+public record ReleaseVersionPublishedEventDto
+{
+    public required Guid ReleaseId {get;init;}
+    public required string ReleaseSlug { get; init; }
+    public required Guid PublicationId { get; init; }
+    public required string PublicationSlug { get; init; }
+    public required Guid PublicationLatestReleaseVersionId { get; init; }   
```

Comment:
Shouldn't this be PublicationLatestPublishedReleaseVersionId given changes made elsewhere in https://github.com/dfe-analytical-services/explore-education-statistics/pull/5694 to ReleaseVersionPublishedEventDto?